### PR TITLE
Add function to show total size of objects

### PR DESCRIPTION
**Pull request by proteusvacuum**
@sravfeyn 
Still looking at memory issues. Here's another tool to help see the size of particular objects.

### CODE REVIEW

**Comment by proteusvacuum** [comment]
@dannyroberts new code buddy

File: dimagi/utils/memory/utils.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
+from __future__ import print_function
+from sys import getsizeof, stderr
+from itertools import chain
+from collections import deque
+try:
+    from reprlib import repr
```
Comment (benrudolph) [inline dimagi/utils/memory/utils.py]: this is odd in a try catch, assuming it's copied though

**Comment by benrudolph** [comment]
sweet!

File: dimagi/utils/memory/utils.py (hunk 1 above)
Comment (proteusvacuum) [inline dimagi/utils/memory/utils.py]: I think its a Python 2 / 3 compatibility thing. But yes, it is copied.

File: dimagi/utils/memory/utils.py [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+except ImportError:
+    pass
+
+
+# From: https://code.activestate.com/recipes/577504/
+def total_size(o, handlers={}, verbose=False):
```
Comment (czue) [inline dimagi/utils/memory/utils.py]: nit: i know copied, but [mutable default arg](http://docs.python-guide.org/en/latest/writing/gotchas/#mutable-default-arguments)

Comment (proteusvacuum) [inline dimagi/utils/memory/utils.py]: Ah. Nice catch. (this was written by a python core dev: https://twitter.com/raymondh, who I guess are fallible too...)

**Pull request merged**